src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceSpecCallbackHandler.java

```diff
@@ -330,16 +330,21 @@ private boolean checkPastDateValidation(LocalDate localDate) {
 
     private CallbackResponse validateAmountPaid(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder<?, ?> updatedCaseData = caseData.toBuilder();
+
         if (caseData.getCcjPaymentDetails() != null
             && YES.equals(caseData.getCcjPaymentDetails().getCcjPaymentPaidSomeOption())) {
             return AboutToStartOrSubmitCallbackResponse.builder()
                 .errors(List.of(PARTIAL_PAYMENT_OFFLINE))
                 .build();
+        } else if (caseData.getFixedCosts() != null && NO.equals(caseData.getFixedCosts().getClaimFixedCosts())) {
```

Comment:
Do we need to put our changes behind the feature flag?

Reply:
@Azam-Hmcts FixedCosts should be null at this stage in prod - I have added fixedCosts (existing field) only to the first page in nonprod to potentially trigger this line:
![image](https://github.com/user-attachments/assets/f17758e9-3b34-40f9-b228-8db03a937859)